gym/vector/async_vector_env.py

```diff
@@ -276,7 +275,7 @@ def close(self, timeout=None, terminate=False):
                     pipe.send(('close', None))
             for pipe in self.parent_pipes:
                 if not pipe.closed:
-                    pipe.recv()
+                    _, success = pipe.recv()
```

Comment:
Was this value supposed to be used anywhere?

Reply:
Seems like it should be used or the change reverted.

Reply:
Fixed it, this was not used anywhere